.github/workflows/StartProgress.yml

```diff
@@ -0,0 +1,39 @@
+name: Start Progress
+
+on:
+  project_card:
+    types: ["moved"]
+
+jobs:
+  assign_to_self:
+    runs-on: ubuntu-latest
+    if: |
+        github.event.changes.column_id.from == 4971951
```

Comment:
```suggestion
        github.event.changes.column_id.from == 4971951 # To Do
```

Reply:
It's not possible to add comments inside `if`. It doesn't work as `//` nor as `#`. See this run:

https://github.com/pavel-mikula-sonarsource/GitHubActionPlayground/actions/runs/400781201

Reply:
For reference, run with `//` is one of the old/early versions: https://github.com/pavel-mikula-sonarsource/GitHubActionPlayground/actions/runs/293418742